### ISSUE TITLE
NVDA misses atom count announcements when holding arrow keys

### ISSUE THREAD

**Issue by vperezcadavid99** [phetsims/beta-decay#64]
**Test device**
Asus Zenbook
**Operating System**
Win 11
**Browser**
Firefox + NVDA
**Problem description**
From https://github.com/phetsims/qa/issues/1441

When holding down the left or right arrow keys, the screen reader fails to announce the updated number of atoms. However, performing single key presses reads the values correctly.

**Steps to reproduce**
1. Turn on NVDA.
2. Navigate to either the Multiple Atoms or Decay Rate screen.
3. Use Tab navigation to focus the addAtomsPanel.
4. Press the Right or Left arrow key once to add atoms. Notice that the number of added atoms is announced correctly.
5. Press and hold down the Right or Left arrow key. The announcements for the updated numbers are missed.

**Visuals**

https://github.com/user-attachments/assets/0d541944-0d6a-4a50-9fd0-ac02cb80ab17

<details>
Troubleshooting information:
!!!!! DO NOT EDIT !!!!!
Name: ‪Beta Decay‬
URL: https://phet-dev.colorado.edu/html/beta-decay/1.0.0-rc.1/phet/beta-decay_all_phet.html
Version: 1.0.0-rc.1 2026-09-11 23:10:20 UTC
Features missing: applicationcache, applicationcache, touch
Flags: pixelRatioScaling
User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/153.0.0.0 Safari/537.36
Language: en-US
Window: 1600x812
Pixel Ratio: 1.7999999523162842/1
WebGL: WebGL 1.0 (OpenGL ES 2.0 Chromium)
GLSL: WebGL GLSL ES 1.0 (OpenGL ES GLSL ES 1.0 Chromium)
Vendor: WebKit (WebKit WebGL)
Vertex: attribs: 16 varying: 30 uniform: 4096
Texture: size: 16384 imageUnits: 16 (vertex: 16, combined: 32)
Max viewport: 32767x32767
OES_texture_float: true
Dependencies JSON: {}
<details/>

**Comment by vperezcadavid99** [comment]
This issue is similar to [https://github.com/phetsims/quantum-wave-interference/issues/487](url)

**Comment by KatieWoe** [comment]
On Jaws I only see this if I reach the end of the spinner and keep holding the arrow at that point. If I stop in the middle of the options the screen reader reads what I would expect.

**Comment by AgustinVallejo** [comment]
@vperezcadavid99 do you hear the atom count context response when you set the sample? Because if so, it's fine if the spinner misses it.

**Comment by vperezcadavid99** [comment]
Ok, in that case, I hear the number in the ctx response when clicking on the addAtoms button.
Closing.
